webpack/components/extensions/HostDetails/Tabs/ContentTab.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import EmptyPage from 'foremanReact/components/common/EmptyState/EmptyStatePattern';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const tabs = [
```

Comment:
Patternfly tables have instilled a bad habit in us of intertwining our JSX markup and our data objects. I think code looks a lot cleaner when the JSX is only at the bottom in render/return, and all the other stuff is here at the top.  Instead of the `map` down on line 46 I feel like it will make this component simpler and easier to read if it's just the literal JSX. So like

```js
return (
    <Tabs
      isSecondary
      activeKey={activeTab}
      onSelect={handleTabClick}
    >
      <Tab title={__('Packages')}...>
        <EmptyPage header="WIP Packages" ... />
      </Tab>
      <Tab title={__('Errata')} ...>
          <EmptyPage header="WIP Errata" ... />
     </Tab>
     ...
    </Tabs>
  );
```
and then you won't need the `tabs` const at all. This way you can easily see what the markup structure on the page will look like too.

Reply:
Addressed